yugabytedb_managed/README.md

```diff
@@ -0,0 +1,70 @@
+# YugabyteDB Managed
+
+## Overview
+
+The [YugabyteDB Managed][1] integration with Datadog provides users with the ability to export metrics from their YugabyteDB Managed clusters into Datadog to monitor and alert on.
+
+## Setup
+- **Note:** This feature is not available for [Sandbox Clusters][5].
+- Additional setup documentation can be found [here][6].
+
+### Installation
+
+To enable the YugabyteDB Managed integration with Datadog:
+
+#### Create a configuration
+1. Within YugabyteDB Managed, go to the **Metrics** tab under the **Integrations** page from the navigation panel on the left side of the page.
+2. Select the **Add Export Configuration** button.
+3. Choose the **Datadog** provider.
+4. Fill in the **API key** and **Site** fields with the corresponding values.
+   - The **API key** is associated with your Datadog organization. If you don't have an API key to use with your YugabyteDB Managed cluster, you need to create one in your [Datadog Organization Settings][2]. For further instructions, see the [Datadog documentation][3].
+   - Your **Site** corresponds to your Datadog Site URL. For more details, see the [Datadog documentation][4].
+5. You can validate the provided configuration by clicking the **Test Configuration** button before selecting **Create Configuration**.
+
+#### Associate a configuration to a cluster
+1. Within YugabyteDB Managed, go to the **Metrics** tab under the **Integrations** page from the navigation panel on the left side of the page.
+2. Find your cluster in the **Export Metrics by Cluster** table.
+3. Open the dropdown for your cluster under the **Export Configuration** dropdown and select the desired configuration.
+4. Wait for the **Export Status** to show `Active`.
+   - **Note:** Your cluster cannot be paused or have another operation in progress when associating a configuration.
+
+### Configuration
+
+Open your Datadog dashboard list and select the `YugabyteDB Managed Overview` dashboard. This integration-bundled dashboard contains pre-built graphs that provide monitoring visibility for your YugabyteDB Managed clusters.
```

Comment:
You can include in the overview that this integration comes with an out-of-the-box dashboard to provide visibility into the YugabyteDB Managed clusters

Reply:
updated.